packages/material-ui-lab/src/Rating/Rating.js

```diff
@@ -34,6 +34,8 @@ export const styles = theme => ({
     fontSize: theme.typography.pxToRem(24),
     color: '#ffb400',
     cursor: 'pointer',
+    // Remove blue highlight
```

Comment:
We can just remove this comment as it obviously causes more confusion. The statement itself already says that the highlight is removed. What color the highlight has is platform/user stylesheet dependent.

Reply:
Sure 👍